src/commands/misc/8ball.js

```diff
@@ -0,0 +1,25 @@
+const { Command, SwitchbladeEmbed, Constants } = require('../../')
+
+module.exports = class EightBall extends Command {
+  constructor (client) {
+    super(client)
+    this.name = '8ball'
+    this.aliases = ['eightball', '8b', 'magicball']
+  }
+
+  run (message, args) {
+    const replies = ['Yes', 'No', 'I don\'t know', 'Ask again later', 'I\'m not so sure', 'You tell me']
```

Comment:
Can we have the 23 standard answers?
https://en.wikipedia.org/wiki/Magic_8-Ball#Possible_answers